src/dev/flang/fe/LibraryModule.java

```diff
@@ -804,7 +804,7 @@ int innerFeaturesNextPos(int at)
 [options="header",cols="1,1,2,5"]
 |====
    |cond.     | repeat | type          | what
-.6+| true  .6+| 1      | byte          | 0FCYkkkk  k = kind, Y = has Type feature (i.e., 'f.type'), C = is intrinsic constructor, F = has 'fixed' modifier
+.6+| true  .6+| 1      | short         | 000000vvvFCYkkkk  k = kind, v = visibility, Y = has Type feature (i.e., 'f.type'), C = is intrinsic constructor, F = has 'fixed' modifier
```

Comment:
please add explanation of `v` here as well, this will be shown in the fum file documentation on flang.dev.